packages/polyfills/package.json

```diff
@@ -6,6 +6,9 @@
   "sideEffects": true,
   "main": "./dist/src/index.js",
   "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "build": "tsc --p tsconfig.json"
```

Comment:
As far as I can tell, this script will never be called.

Reply:
It's to maintain consistency with other packages in the repo. It's for cases when you'd want to build just one single package.